test/db.test.js

```diff
@@ -1,48 +1,34 @@
 // tests for database/index.js
 require('dotenv').config();
-var db = require('../database/index');
-var expect = require('chai').expect;
-
-/*
- * https://mochajs.org/#arrow-functions
- *
-   Passing arrow functions (“lambdas”) to Mocha is discouraged.
-   Lambdas lexically bind this and cannot access the Mocha
-   context. If you do not need to use Mocha’s context, lambdas
-   should work. However, the result will be more difficult to
-   refactor if the need eventually arises.
- *
- */
+const db = require('../database/index');
 
 // test main database function
 // get trail data from trail_id
-describe('getAtrail() db function', function () {
-
-  context('with trail ID argument', function () {
-    it('should return results object to callback with all required keys', function (done) {
+test('getAtrail returns results object to callback with all required keys', (done) => {
+  // TODO: change to getTrail
 
-      db.getAtrail(1, function (trailResult) {
-        expect(trailResult[0])
-          .to.be.a('object')
-          .and.to.have.all.keys('trail_id', 'trail_name', 'distance', 'distance_units', 'elevation_gain', 'elevation_units', 'description', 'route_type', 'difficulty', 'general_area', 'origin');
-        done();
-      });
-    });
+  db.getAtrail(1, function (trailResult) {
+    expect(trailResult[0]).toBeInstanceOf(Object);
```

Comment:
this is fine , and knitpickyness, but i would maybe simplify this with an array of properties and a `foreach` over to check what you need to check. But just a comment no change needed

``` js
['trail_id', 'trail_name', ...].forEach((prop) => { expect(trailResult[0]).toHaveProperty(prop) })
```

Reply:
Great tip, thanks. That would be a better test of the results object, thanks. I'll add an issue to my repo to refactor this test. Thanks!